tests/test_sharding.py

```diff
@@ -4,13 +4,26 @@
 import jax.random as jr
 import jax.sharding as jshard
 import jax.tree_util as jtu
+import pytest
 
 
+# Info on the skips: on CPU-only, JAX can simulate mulitple devices
+# but as soon as you have a GPU(s), JAX will use those, so if you have
+# only a single GPU (i.e. a single device) it makes no sense to "test"
+# sharding
+
+
+@pytest.mark.skipif(
```

Comment:
Maybe we could make this condition be CPU-vs-GPU, so that a single-CPU would explicitly fail this test rather than silently passing.

I don't want these tests to accidentally end up always being skipped because we misconfigure the CPU.